elastic/kibana
# [Canvas/Reporting] Migrate Canvas to V2 reporting

## Summary

This contribution migrates Canvas to use V2 PNG report generation (powered by locators). As a result a new canvas locator was created.

New export types were introduced in https://github.com/elastic/kibana/pull/101048.


## Screenshot

<img width="1118" alt="Screenshot 2021-08-24 at 17 05 49" src="https://user-images.githubusercontent.com/8155004/130641406-e5069c5a-26d3-48df-b6fa-12c72d2a76b3.png">

## How to test

1. Create a new space in Kibana
2. Create a new workpad (using sample data)
3. Add several new pages
4. Export as PDF
5. Check the result

## Checklist
